scopes/design/ui/cli-snippet/cli-snippet.compositions.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { CliSnippet } from './index';
+
+const content = `this is some cli content\n which should be rendered properly as html`;
+
+export const CliSnippetExample = () => {
+  return <CliSnippet content={content} />;
+};
```

Comment:
Looks good.

Reply:
I'll add Height with canvas API to limit the Height of the composition in the Overview tab.